src/components/savedmemes/SavedMemes.tsx

```diff
@@ -18,10 +20,24 @@ class SavedMemes extends React.Component<SavedMemesProps, SavedMemesState>{
 		}
 	}
 
+	displaySavedMemes = () => {
```

Comment:
Do we want to add in a conditional statement here to render a message to the DOM if there are no saved memes yet? Just so that the user doesn't end up on a blank page if the savedMemes array is empty?